### DISCUSSION TITLE
Should I avoid docker for deployment

### DISCUSSION THREAD

**Discussion by DavidKaub** [ckan/ckan#7446] [Q&A]
Hi everyone,

we used a CKAN docker compose installation in the past, but had some performance issues where the frontend would basically freeze in some situations, and we had to restart the container.

We're now using a CKAN source installation deployed using nginx as described in the docs. Performance is good now, but we're having issues uploading bigger files, as described in [another discussion](https://github.com/ckan/ckan/discussions/7445). Now I'm thinking of migrating back to docker, since we had not had those problems there. Is there any reason why one option is superior over the other for deployment, running on ubuntu-server 22.04?

Greetings

**Comment by ghost** [comment]
I used this docker version of CKAN to base our docker build on.  https://github.com/okfn/docker-ckan  

it works similarly to the main docker build but without NGINX (as its docker we opted for IIS).   

It seems to work well with no performance issues so far - 224 datasets.
